app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -281,6 +285,31 @@ class BrowserTabViewModel(
         skipHome = false
     }
 
+    suspend fun onUserSubmittedAutocomplete(suggestion: AutoCompleteApi.AutoCompleteSuggestion) {
+        onUserSubmittedQuery(suggestion.phrase)
+
+        val hasBookmarks = withContext(Dispatchers.IO) { bookmarksDao.hasBookmarks() }
```

Comment:
personal preference here, but i feel like putting this on one line makes it harder to see what is actually being executed. when on separate lines, the indentation helps me quickly scan and parse what code is happening. e.g, 👇

    withContext(Dispatchers.IO) { 
        bookmarksDao.hasBookmarks() 
    }

Reply:
Done!